egs/librispeech/ASR/distillation_with_hubert.sh

```diff
@@ -0,0 +1,95 @@
+stage=$1
+export CUDA_VISIBLE_DEVICES="2,3,4,5"
+
+
+if [ $stage -eq 0 ]; then
+  # Preparation stage.
+
+  # Install fairseq according to:
+  # https://github.com/pytorch/fairseq
+  # when testing this code:
+  # commit 806855bf660ea748ed7ffb42fe8dcc881ca3aca0 is used.
+  #
+  # Install quantization toolkit:
+  # pip install git+https://github.com/danpovey/quantization.git@master
+  # when testing this code:
+  # commit c17ffe67aa2e6ca6b6855c50fde812f2eed7870b is used.
```

Comment:
Please add some checks to ensure that fairseq and quantization are installed before preceding.

Reply:
> some checks
could you give some examples how to check it in shell script?

Reply:
```bash
has_fairseq=$(python3 -c "import importlib; print(importlib.util.find_spec('fairseq') is not None)")
if [ $has_fairseq == 'False' ]; then
  echo "Please install fairseq before preceding"
  exit 1
fi
```

Reply:
Thanks.